terraform/panther_gcs_transport_type_infra/panther.tfvars

```diff
@@ -0,0 +1,21 @@
+# Basic Info
+project_id = "project-id" # ID (name) of your GCP project
+gcp_region = "gcp-region" # GCP Region that your infrastructure will live
+gcp_zone   = "gcp-zone"   # GCP Availability Zone that your infrastructure will live
+
+# Panther will pull data from this GCS bucket by way of Pub/Sub Notifications
+# If your bucket already exists you would need to either import the resource in this terraform
+# project or remove the `resource` definition from the `main.tf` file.
+bucket_name         = "bucket-name"
+topic_name          = "topic-name"
+subscription_id     = "subscription-id"
+gcs_bucket_location = "gcs-bucket-location"
+
+# Prefixes that notifications will be setup for. Data will need to be added within thoses prefixes
+# You need to edit the following to include your subdirectories.
```

Comment:
Is `sub-directory` the correct term here? I understand that GCS uses the prefix term too: https://cloud.google.com/storage/docs/samples/storage-list-files-with-prefix

Reply:
Renamed to prefixes